src/assets/scss/global/typography.scss

```diff
@@ -0,0 +1,25 @@
+@import 'settings';
+
+h1 {
+	font-weight: bold;
```

Comment:
This can be declared in the header-style for "small" and will apply to h1's on all screen sizes.

Reply:
Actually, the $header-style thing currently only supports font-size, line-height, margin-top, and margin-bottom. I originally had the font-weight in there, but it was not being applied.

Reply:
ahh gotcha...sorry about that. wishful thinking

Reply:
Yeah... though we should still consider moving the header styles somewhere else so they can be edited in one place